pvlib/tracking.py

```diff
@@ -244,6 +247,16 @@ def get_irradiance(self, surface_tilt, surface_azimuth,
         ghi = self._validate_per_array(ghi, system_wide=True)
         dhi = self._validate_per_array(dhi, system_wide=True)
 
+        if albedo is None:
+            try:
+                albedo = kwargs.pop('albedo')
+            except KeyError:
```

Comment:
I think it's not possible for `kwargs` to contain `albedo` here since it's already a named parameter to this function.